docs/source/common_issues.rst

```diff
@@ -173,6 +173,82 @@ and difficult-to-predict failure modes and could result in very
 confusing error messages. The tradeoff is that you as a programmer
 sometimes have to give the type checker a little help.
 
+.. _version_and_platform_checks:
+
+Python version and system platform checks
+-----------------------------------------
+
+Mypy supports the ability to perform Python version checks and platform
+checks (e.g. Windows vs Posix), ignoring code paths that won't be run on
+the targeted Python version or platform. This allows you to more effectively
+typecheck code that supports multiple versions of Python or multiple operating
+systems.
+
+More specifically, mypy understands ``if/elif`` statements of the following
+forms:
```

Comment:
For the examples, do you want me to try and add some actual code within the if statements, or should I just keep the comments and leave them otherwise blank?